### ISSUE TITLE
Undefined method `untaint` with Ruby 3.2

### ISSUE THREAD

**Issue by akaspick** [yujinakayama/transpec#141]
This is likely an issue with Ruby 3+ as taint/untaint have been removed.

When attempting to run transpec, I get the following error:
```
> transpec -f
undefined method `untaint' for "/Users/user/Documents/Development/project":String
```

**Comment by neerajkumar61** [comment]
Try this 
rm -f Gemfile.lock
bundle

**Comment by jasonkarns** [comment]
I don't think that blowing away an entire lockfile is safe or a good resolution to the problem.

Especially if transpec is being used "globally" (ie, outside one's bundle; as is recommended on the readme).

Any additional thoughts here?
